MvvmCross/Platforms/Android/Core/MvxAndroidSetup.cs

```diff
@@ -230,7 +230,10 @@ protected virtual IEnumerable<Assembly> ValueConverterAssemblies
 
         protected virtual IDictionary<string, string> ViewNamespaceAbbreviations => new Dictionary<string, string>
         {
-            { "Mvx", "MvvmCross.Platforms.Android.Views" }
+            { "Mvx", "mvvmcross.platforms.android.views"
+            },
+            { "Mvx", "mvvmcross.platforms.android.binding.views"
```

Comment:
Having 2 items with the same key (Mvx) won't work in a dictionary

Reply:
Aah true, but don't you need both namespaces? And wouldn't `mvvmcross.platforms.android.binding.views` be more important to keep?

Reply:
Changing the namespace to be `binding.views` will result in a breaking change, won't it? 🤔

Reply:
Well, i think it might not be working at all right now. I always use full names, maybe someone can give this a quick test?